tmail-backend/tmail-third-party/open-ai/src/main/java/com/linagora/tmail/mailet/Llm.java

```diff
@@ -0,0 +1,28 @@
+package com.linagora.tmail.mailet;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+public enum Llm {
+    OPEN_AI("openai");
```

Comment:
We only support openai as of today.

Let's not over anticipate future need, remove this constant and related configuration?